app/workers/push_notifications/cleanup_worker.rb

```diff
@@ -0,0 +1,18 @@
+module PushNotifications
+  class CleanupWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform
+      redis = Redis.new
```

Comment:
Do we have a persistent connection pool for this Redis instance somewhere so that we don't have to spin up a new one for this operation?

Also, which Redis instance does this connect to?

Reply:
I couldn't find any kind of redis pooling in the codebase. As far as which Redis instance this is connecting to: default based on this code! I'll add an environment variable for at least configure a redis instance for holding this data.

Definitely down to use any kind of pooling you'd recommend @jgaskins, but if it's new to the codebase maybe that should be in a different PR?

Reply:
@joshpuetz Interesting, it looks like this is our first direct use of Redis. Connection pooling is not a huge deal for this particular instance since Redis is reasonably efficient with opening and closing connections and it's only running 3x/day anyway, so I'm absolutely onboard with not blocking the PR for that. :-)